Simplified/SignInLogic/NYPLSignInBusinessLogic.swift

```diff
@@ -287,7 +287,11 @@ class NYPLSignInBusinessLogic: NSObject, NYPLSignedInStateProvider, NYPLCurrentL
       switch result {
       case .success(let responseData, _):
         #if FEATURE_DRM_CONNECTOR
-        self.drmAuthorizeUserData(responseData, loggingContext: loggingContext)
+        if NYPLSettings.shared.useBetaLibraries {
```

Comment:
here we are bypassing the extraction of DRM authorization info for beta libraries, which doesn't seem quite right -- AFAIK beta libraries are supposed to work the same way as prod ones (kinda like a staging system). It has also been working fine for SimplyE.  Is this maybe meant to be just for Open eBooks? Or is it a temporary workaround?

Reply:
It's a temporary workaround until the PR for https://jira.nypl.org/browse/SIMPLY-3607 gets merged in

Reply:
Removed.